src/lib.rs

```diff
@@ -134,19 +134,16 @@ impl<S: Hash + Sync + Send + Clone + Default + Reflect + FromReflect + TypePath
 }
 
 fn map_input_zones_from_ui_nodes<S: StickIdType>(
-    interaction_areas: Query<(&Transform, &Node), With<TouchStickInteractionArea>>,
-    mut sticks: Query<&mut TouchStick<S>>,
+    mut interaction_areas: Query<
+        (&mut TouchStick<S>, &GlobalTransform, &Node),
+        With<TouchStickInteractionArea>,
+    >,
```

Comment:
This change forces the interaction area and stick to be on the same entity. The idea was that you should be able to have multiple interactable zones for the same stick to block out other ui etc. But the feature is only half done.

With this change, the marker `TouchStickInteractionArea` serves no purpose and could be removed.

However, this does kind of solve the issue with multiple sticks, so maybe it's ok for now.